python/paddle/fluid/layers/nn.py

```diff
@@ -6446,15 +6481,15 @@ def affine_grid(theta, out_shape, name=None):
                         [x_14, x_15, x_16]]
                        [[x_21, x_22, x_23]
                         [x_24, x_25, x_26]]]
-
+      
```

Comment:
Are you sure your pylint is working correctly?

Reply:
It made by my CLION, I have reverted it.